numba/tests/test_np_functions.py

```diff
@@ -14,12 +14,19 @@
 
 enable_pyobj_flags = Flags()
 enable_pyobj_flags.set("enable_pyobject")
-
 no_pyobj_flags = Flags()
+no_pyobj_flags.set("nrt")
+
+no_pyobj_flags = [no_pyobj_flags]
+enable_pyobj_flags = [enable_pyobj_flags]
+all_flags = enable_pyobj_flags + no_pyobj_flags
```

Comment:
Not sure about this, but I think testing in object mode has started to become pedantic a long time ago... after all, we're just asking the interpreter to call a function, so there's not much opportunity for misbehaviour on a particular function if other function calls work ok :-) In other words, I'd say you probably shouldn't bother changing this, and focus on nopython mode.